stable/distributed-tensorflow/values.yaml

```diff
@@ -5,21 +5,27 @@ worker:
   number: 2
   podManagementPolicy: Parallel
   image:
-    repository: cheyang/distributed-tf
-    tag: 1.6.0
+    repository: dysproz/distributed-tf
+    tag: 1.7.0
     pullPolicy: IfNotPresent
   port: 9000
-  # gpuCount: 2
 ps:
   number: 2
   podManagementPolicy: Parallel
   image:
-    repository: cheyang/distributed-tf
-    tag: 1.6.0
+    repository: dysproz/distributed-tf
+    tag: 1.7.0
     pullPolicy: IfNotPresent
   port: 8000
 # optimize for training
 hyperparams:
   batchsize: 20
   learningrate: 0.001
   trainsteps: 0
+volumes:
+  - name: logs-volume
+    hostPath:
```

Comment:
This should be converted to generic storage PVC with RWX with storage provider and so on (with extravolumes option). Not everyone has hostpath available in their cluster.

See example under fluentd chart - https://github.com/helm/charts/blob/master/stable/fluentd/templates/deployment.yaml

Reply:
I'll leave it as it was without mounting volumes. You're right that logs should be left as it was. I'll only leave the change with nice formatting of logs in order to simplify debugging.